categoriesChoos.js

```diff
@@ -0,0 +1,69 @@
+var restaurantsToShow = inicializeRestaurantToShow();
+var idChoosenCategory = '';
+
+$(document).ready(function() {
+  idChoosenCategory = splittedLocationHash();
+  restaurantsToShow = inicializeRestaurantToShow();
+  showCategory();
+  moment.locale('pl');
+  $('#singInButton').on('click', function() {
```

Comment:
let's handle signup in signup.js file (chooseCategory wouldn't be my first choice to search in for signup modal code calls ;) ) 

btw. we have a typo in file name ;) should be `Choose` or even better `Choice`